orchestrator/cosmos_gravity/src/send.rs

```diff
@@ -149,8 +149,8 @@ pub async fn send_messages(
     args.fee.gas_limit = cmp::max(gas_limit as u64, 500000 * messages.len() as u64);
 
     // compute the fee as fee=ceil(gas_limit * gas_price)
-    let fee_amount: f64 = args.fee.gas_limit as f64 * gas_price.0;
-    let fee_amount: u64 = fee_amount.abs().ceil() as u64;
+    let fee_amount = (args.fee.gas_limit as u128).checked_mul(gas_price.0 as u128)
```

Comment:
Since `gas_price` is an f64, isn't this truncating the decimals by casting as an integer? If `gas_price` is less than 1 this will result in 0 fees.

Reply:
yes you are correct, casting a f64 to integer will result truncate of all the decimal value.
ex
0.86 will be casted to 0

Is SOM using decimal gas price?